eng/scripts/vcpkg.ps1

```diff
@@ -0,0 +1,67 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $Ref = 'master',
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $Dependencies,
+
+    [Parameter()]
+    [string] $TargetPath ="$env:TEMP/$([guid]::NewGuid())",
+
+    [Parameter()]
+    [switch] $CI = ($null -ne $env:SYSTEM_TEAMPROJECTID)
```

Comment:
We have been using $devOpsLogging as the variable name in a few places like https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/Verify-Links.ps1#L7. I do like that you defaulted it from a DevOps env variable instead of us needing to explicitly pass it.

@chidozieononiwu I believe you have an issue on your backlog to move the logging into our common scripts when you do you should use this pattern. I'd prefer not to use the name `$CI` because that is a little ambiguous but the pattern of initializing we should use.